docs/integrations/dbt.md

```diff
@@ -85,7 +90,7 @@ For example, the SQL `WHERE` clause with the "ds" column goes in a jinja block g
 > {% endif %}
 ```
 
-`{{ start_ds }}` and `{{ end_ds }}` are the jinja equivalent of SQLMesh's `@start_ds` and `@end_ds` predefined time variables. See all [`predefined time variables`](../concepts/macros.md#predefined-variables) available in jinja.
+`{{ start_ds }}` and `{{ end_ds }}` are the jinja equivalents of SQLMesh's `@start_ds` and `@end_ds` predefined time macro variables. See all [predefined time variables](../concepts/macros.md#predefined-variables) available in jinja.
 
```

Comment:
Let's add a section that introduces sqlmesh's additional incremental fields and how to use them in a dbt project. (They can be set in the model config or the model config "meta" dict).

- batch_size
- lookback